src/lib/Scenes/Artwork/Components/ArtworkExtraLinks.tsx

```diff
@@ -22,6 +22,14 @@ export class ArtworkExtraLinks extends React.Component<ArtworkExtraLinksProps> {
     SwitchBoard.presentNavigationViewController(this, `/inquiry/${artwork.slug}`)
   }
 
+  handleReadOurAuctionFAQsTap = () => {
+    // FIXME: Add auction FAQs navigation here
```

Comment:
Might be good to add an alert here that says "TODO" or somesuch so people doing QA before this is implemented don't get confused.